grafana-plugin/src/containers/EscalationChainSteps/EscalationChainSteps.tsx

```diff
@@ -80,13 +91,19 @@ const EscalationChainSteps = observer((props: EscalationChainStepsProps) => {
             return null;
           }
 
+          const extraProps: Partial<EscalationPolicyProps> = {};
```

Comment:
I see a lot of logic inside loop, that not very readable, can we move preparing props for  `EscalationPolicy` outside of loop, e.g create some helper...